diktat-rules/src/test/resources/test/paragraph3/boolean_expressions/DistributiveLawExpected.kt

```diff
@@ -21,4 +21,19 @@ fun some() {
     if (a > 5 && (b > 6 || c > 7 || d > 8)) {
 
     }
+
+    // long case
+    if (a > 5 && ((b > 6 && z > 3) || (c > 7 && y > 4) || (d > 8 && w > 5))) {
```

Comment:
First I wanted to suggest not to use braces with `&&` as `a && b || c && d` is the same to `(a && b) || (c && d)`, but now I think that your version is better